src/main/java/de/unistuttgart/overworldbackend/service/PlayerTaskStatisticService.java

```diff
@@ -134,30 +140,27 @@ public PlayerTaskStatisticDTO submitData(final PlayerTaskStatisticData data) {
     );
   }
 
-  private void calculateCompletedDungeons(final PlayerStatistic playerstatistic) {
+  private void isDungeonCompleted(Dungeon dungeon, PlayerStatistic playerStatistic) {
```

Comment:
the name implicates that the return value is a boolean (and typically no logic stuff is made, at least no update queries). So I recommend renaming the function and maybe return a boolean.

Reply:
You also forgot the `final`s